packages/trip-form/src/TripOptions/styled.ts

```diff
@@ -136,14 +137,21 @@ export const OptionButton = styled.button`
   }
 `;
 
-export const OptionLabel = styled.div``;
+export const OptionLabel = styled.div`
+  flex: 2;
+`;
 
 export const OptionIcon = styled.div`
   > svg {
     margin-bottom: 0px;
   }
 `;
 
+export const OptionImage = styled.img`
+  max-height: 20px;
+  width: 50px;
+`;
```

Comment:
Hey @miles-grant-ibigroup, how would I go about setting the .svg fill color for the images I'd be using.  Basically, I think the current design would call for all the .svg button icons to be white.

Reply:
FWIW, created a set of .svgs with the color swited to white, so that the designers can get their hands on this.  But probably want to do this with a 'fill' prop.  Further, in the future there *could* be a light and dark mode in the app, and so programmatically setting the fill color (as opposed to specifying the color in a config file) would be ideal.  Stuff to think about...  Short-term, we could just the -white.svg image set if need be...

Reply:
p.s., the aforementioned white SUM .svg icons in TORA: https://labs.trimet.org/home/planner-mode-settings
(will be using this to get designer / business owner approval of the mode selector form).

Reply:
Since `OptionImage` isn't guaranteed to contain an svg, I'm weary of creating a fill prop for it. I believe using an inline style prop should be able to achieve this (`style={{fill: 'white'}}`). However, if that feels dirty to you then adding a fill prop should be doable!

What do you think?